frontend/src/App.js

```diff
@@ -0,0 +1,93 @@
+import 'App.scss';
+
+import React, {Component} from 'react';
+
+import API from 'Utilities/API';
+
+type AppPageProps = {};
+type AppPageState = {
+    user: {};
+    identity: {};
+    dataProduct: {};
+};
+
+class App extends Component <AppPageProps, AppPageState> {
+    constructor(props: AppPageProps) {
+        super(props);
+        this.state = {};
+        this.onLogin = this.onLogin.bind(this);
+        this.onLogout = this.onLogout.bind(this);
+    }
+
+    async componentDidMount() {
+        const { ok, data } = await API.me();
+        if (ok) {
+            this.setState({ user: data }, async () => {
+                const { ok, data } = await API.getIdentity(this.state.user['@id']);
+                if (ok) {
+                    this.setState({ identity: data }, async () => {
+                        const { ok, data } = await API.fetchDataProduct('prh-business-identity-data-product', { businessId: '2980005-2' });
```

Comment:
Could you extract the `'2980005-2'` to just some config variable or something so it's less of a magic number in the middle of the code?